### DISCUSSION TITLE
[Bug report]: Fresh install of Firefly 1.5.1 and backing up from seed phrase shows total balance 0

### DISCUSSION THREAD

**Discussion by dbforma** [iotaledger/firefly#2990] [Help & Support]
### Platform

Desktop

### Version

1.5.1

### Description

Fresh install of Firefly 1.5.1 and backing up from seed phrase shows total balance 0

### Expected behaviour

Should import wallets and amount correctly.

### Actual behaviour

I had a income wallet that total balance was actually 0 but I had two additional internal wallets that I've been using to stake, therefor had balance. This was around 1.3.1 - I think when staking initially started. Interestingly when I backed up from seed the only wallet I saw was a default wallet (Wallet 1) with balance 0 and an address I have never seen before. I tried to search for balance and it found some wallets but the balance still shows 0 so that didn't work either. In Discord was suggested to create two additional wallets but as I have never used this new address of "Wallet 1" before that was not possible.

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

1. First wallet should have a balance of 0, other wallets have balance
2. Fresh Install Firefly
3. Backup from seed phrase
 

### Operating System

Linux

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

Previously staked

### Crash Status

None of the above

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nicole-obrien** [comment]
@dbforma please can you try and use the balance finder in settings > advanced > balance finder

**Comment by dbforma** [comment]
Did that as described and it found wallets but balance 0

Am 20. Apr. 2022, 14:05, um 14:05, nicole-obrien ***@***.***> schrieb:
***@***.*** please can you try and use the balance finder in settings >

**Comment by nicole-obrien** [comment]
If you use the balance finder multiple times, it will search more addresses for each wallet and you might find the balance.
